nltk/test/binary_fixt.py

```diff
@@ -0,0 +1,11 @@
+from nltk.internals import find_binary
+
+
+def check_binary(binary):
+    import pytest
+
+    try:
+        find_binary(binary)
+    except:
+        pytest.skip("Skipping test because the {binary} binary was not found")
```

Comment:
Missing the `f` prefix to the string here.

Reply:
Ok, fixed.